modules/trusted-firmware-m/Kconfig.tfm

```
@@ -198,13 +198,20 @@ config TFM_IMAGE_VERSION_NS
 	  versions of both the secure firmware and non-secure firmware. This sets
 	  the non-secure firmware's version for rollback prevention.
 
+config TFM_BL1
+	bool "Use BL1 immutable bootloader from TFM"
+	default n
+	help
+          TFM provides a reference immutable bootloader, BL1.
+          This config adds BL1 to the build - built via TFM's build system.
+
 config TFM_BL2
-	bool "Add MCUboot to TFM"
+	bool "Use BL2 bootloader from TFM"
 	depends on !TFM_BL2_NOT_SUPPORTED
 	default y
 	help
-	  TFM is designed to run with MCUboot in a certain configuration.
-	  This config adds MCUboot to the build - built via TFM's build system.
+          TFM provides a reference second-stage flash bootloader, BL2.
+          This config adds BL2 to the build - built via TFM's build system.
```

I think some further clarification is useful here:

```suggestion
          TFM provides a reference second-stage flash bootloader, BL2.
          This bootloader is usually based on MCUBoot, and is used to
          verify firmware updates for subsequent images in the chain
          of trust. This config adds BL2 to the build - built via TFM's
          build system.
```

---

Thanks. :)

Added a commit with your review comments.